modules/ui/commit.js

```diff
@@ -25,7 +25,11 @@ var readOnlyTags = [
     /^host$/,
     /^locale$/,
     /^warnings:/,
-    /^resolved:/
+    /^resolved:/,
+    /^closed:note$/,
```

Comment:
Hmm can we actually move this behavior to a different PR? It's a good idea but I see it as its own new thing.